prime-router/src/main/kotlin/cli/tests/AuthTests.kt

```diff
@@ -80,7 +80,7 @@ class OktaAuthTests : CoolTest() {
 
     override suspend fun run(environment: Environment, options: CoolTestOptions): Boolean {
         var passed = true
-        val org1 = historyTestOrgName
+        val org1 = historytestOrgName
```

Comment:
I concur with @oslynn's suggestion to maintain the variable naming convention of `historyTestOrgName`